README.md

```diff
@@ -24,6 +24,10 @@ $ make prepare.ir
 Enter account NTrezR3C4X8aMLVg7vozt5wguyNfFhwuFx password >
 Sent invocation transaction fa6ba62bffb04030d303dcc95bda7413e03aa3c7e6ca9c2f999d65db9ec9b82c
 ```
+Also you can add self-signed node (`s04.neofs.devenv`) certificate to truststore:
+```
+$ sudo make prepare.storage
```

Comment:
I wouldn't be doing that, see `make hosts` for example, it provides you with data but you decide how to use it (of course some instruction would be helpful here).

Reply:
Maybe it's just me though, @realloc?